### PR TITLE
Basic test of osm2pgsql-replication

### PR DESCRIPTION
**Pull request by JakobMiksch**
- add basic test for `osm2pgsql-replication` by checking if replication table can be created in public schema and custom schema
- the test data `tests/data/liechtenstein-2013-08-03.osm.pbf` needed to be extended by a (dummy) replication header, because  `osm2pgsql-replication` requires that to run properly:
    
    ```
    osmosis_replication_base_url=http://example.com/europe/liechtenstein-updates
    osmosis_replication_sequence_number=9999999
    ```

- the function `def run_osm2pgsql` was modified to allow constant injection for the created CLI arguments
- run this test with: `pg_virtualenv behave command-line/replication.feature`

### CODE REVIEW

**Comment by JakobMiksch** [comment]
Meanwhile I managed to test the actual effect of the script. Of course there are many more tests possible. But I stumbled upon the problem that the test data `liechtenstein-2013-08-03.osm.pbf` does **not** contain the replication header `osmosis_replication_base_url` that is needed for testing the script. 

For my local testing I use the freshly downloaded `liechtenstein-latest.osm.pbf`.

@joto @lonvia is it possible to add a replication header to the existing file, or does it make sense to add a new test file, or do you have any other idea?

thanks in advance

File: tests/bdd/command-line/replication.feature [hunk 1]
```diff
@@ -0,0 +1,32 @@
+Feature: Tests for the osm2pgsql-replication script
+
+    Scenario: Replication can be initialiased
```
Comment (JakobMiksch) [inline tests/bdd/command-line/replication.feature:3]:
```suggestion
    Scenario: Replication can be initialised
```

File: tests/bdd/command-line/replication.feature [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            """
+        When running osm2pgsql pgsql with parameters
+            | --slim |
+
+        And running osm2pgsql-replication
+            | init | --osm-file={TEST_DATA_DIR}/liechtenstein-2013-08-03.osm.pbf |
+
+        Then table planet_osm_replication_status has 1 row
+
+    Scenario: Replication can be initialiased in different schema
```
Comment (JakobMiksch) [inline tests/bdd/command-line/replication.feature:17]:
```suggestion
    Scenario: Replication can be initialised in different schema
```

**Comment by JakobMiksch** [comment]
Hi @lonvia 
I finally manged to add a replication header to the test OSM file and also adapted the CI.

**Comment by JakobMiksch** [comment]
@lonvia I have pulled the latest changes from `master` and fixed the conflict

**Comment by lonvia** [comment]
@JakobMiksch Can you please rebase onto master and then force-push on this branch so that we have a nice linear commit history?

**Comment by JakobMiksch** [comment]
@lonvia sure, I have done it.

**Pull request merged**